src/ray/util/tests/BUILD

```diff
@@ -237,7 +237,10 @@ ray_cc_test(
         "@com_google_googletest//:gtest_main",
     ],
     size = "small",
-    tags = ["team:core"],
+    tags = [
+        "team:core",
+        "no_tsan", # Fails on CI, but doesn't reproduce for 10000 runs.
```

Comment:
Seems it's consistently reproducible on CI? If you cannot reproduce locally, try use the CI env.

Reply:
I don't think there's a data race